File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.java
class Solution {
    public int totalMoney(int n) {
        int sum1=0;
        int sum2=0;
        int m=n%7;
        int o=n/7;
        for(int k=o+1;k<=o+m;k++){
            sum1+=k;
        }
        if(n>7){
            for(int i=1;i<=o;i++){
                for(int j=i;j<i+7;j++){
                    sum2+=j;
                }
            }
        }
        return sum1+sum2;
    }
}

File: 1716-calculate-money-in-leetcode-bank/1716-calculate-money-in-leetcode-bank.py
class Solution:
    def totalMoney(self, n: int) -> int:
        sum=0
        m=n%7
        o=n//7
        for i in range(o+1,o+m+1):
            sum+=i
        if n>7:
            for j in range(1,o+1):
                for k in range(j,j+7):
                    sum+=k
        return sum